demo/site/src/blocks/PageContentBlock.tsx

```diff
@@ -25,6 +24,7 @@ const supportedBlocks: SupportedBlocks = {
     textImage: (props) => <TextImageBlock data={props} />,
     damVideo: (props) => <DamVideoBlock data={props} />,
     youTubeVideo: (props) => <YouTubeVideoBlock data={props} />,
+    vimeoVideo: (props) => <VimeoVideoBlock data={props} />,
```

Comment:
In Admin and API you remove all video blocks from the page content block, so it should be removed here as well.

Reply:
indeed, missed that one, deleted them 👍